package ru.eng.ai.exception

object ChatClosedException : Exception()